core/imt/src/TThreadExecutor.cxx

```diff
@@ -139,37 +136,33 @@ namespace ROOT {
    ///
    /// At construction time, TThreadExecutor automatically enables ROOT's thread-safety locks as per calling
    /// ROOT::EnableThreadSafety().
-   TThreadExecutor::TThreadExecutor(UInt_t nThreads)
-   {
-      ROOT::EnableThreadSafety();
```

Comment:
why did you remove the call to `EnableThreadSafety()`?

Reply:
nevermind, `InitGlobalTaskArena` calls it now

Reply:
I moved it to the task_arena initialization